src/Compilers/CSharp/Portable/Symbols/Source/SourceFixedFieldSymbol.cs

```diff
@@ -122,11 +122,7 @@ public sealed override int FixedSize
                     if (Interlocked.CompareExchange(ref _fixedSize, size, FixedSizeNotInitialized) == FixedSizeNotInitialized)
                     {
                         this.AddDeclarationDiagnostics(diagnostics);
-                        if (state.NotePartComplete(CompletionPart.FixedSize))
-                        {
-                            // FixedSize is the last completion part for fields.
-                            DeclaringCompilation.SymbolDeclaredEvent(this);
```

Comment:
Yes, sort of. It is handled where the completion part `ConstantValue` is set in `SourceFieldSymbolWithSyntaxReference` in `SourceFieldSymbol.cs`. `SourceFieldSymbolWithSyntaxReference` is the only direct subclass of the abstract class `SourceFieldSymbol` (perhaps we should consider merging them).